index.js

```diff
@@ -1,41 +1,113 @@
 var create = require('webgl-context')
-var getPixels = require('canvas-pixels').get3d
-var triangle = require('a-big-triangle')
 var xtend = require('xtend')
 var assign = require('xtend/mutable')
+var glExt = require('webglew')
+var Framebuffer = require('gl-fbo')
+var Shader = require('gl-shader')
 
-module.exports = function(opt) {
-    opt = xtend({ 
-        width: 1, 
+
+module.exports = function (shader, opt) {
+    if (!opt) {
+        //just options
+        if (typeof shader === 'object' && !shader.fragShader) {
```

Comment:
Small thing here – I think it's best to reduce the peer dependency on `fragShader` and `gl-shader` and just assume a more generic interface that is basically `{ gl, bind() }` which is what we had before. :smile: But I think this part will go away if we keep the options simple as they were before.